docs/marketplace/dev/feature-walkthroughs/202204.0/marketplace-shopping-lists-feature-walkthrough.md

```diff
@@ -0,0 +1,43 @@
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Marketplace Shopping Lists feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-shopping-lists-feature-integration.html) | [Retrieving Marketplace Shopping Lists](/docs/marketplace/dev/glue-api-guides/{{page.version}}/merchants/retrieving-marketplace-shopping-lists.html) | [File details: file-details-product-offer-shopping-list.csv](/docs/marketplace/dev/data-import/{{page.version}}/file-details-product-offer-shopping-list.csv.html) |                                                                                                                                                             |
+|                                                                                                                                                                         |                                                                                                                                                      |                                                                                                                                                                    | [Managing shopping lists](/docs/marketplace/dev/glue-api-guides/{{page.version}}/marketplace-shopping-lists/managing-marketplace-shopping-lists.html)       | 
+|                                                                                                                                                                         |                                                                                                                                                      |                                                                                                                                                                    | [Managing shopping list items](/docs/marketplace/dev/glue-api-guides/{{page.version}}/marketplace-shopping-lists/managing-marketplace-shopping-list-items.html) | 
```

Comment:
- glue API IG link is missing
- please carefully recheck all the linked docs exist in related PRs